tests/ut/ir/parser/test_remote_load.py

```
@@ -0,0 +1,290 @@
+            def kernel(
+                self,
+                data: pld.DistributedTensor[[64], pl.FP32],
+                peer: pl.Scalar[pl.INT32],
+            ) -> pl.Tensor[[64], pl.FP32]:
+                t = pld.tile.remote_load(data, peer=peer, offsets=0, shape=[32])  # type: ignore[arg-type]  # noqa: F841
+                return data  # type: ignore[return-value]
+
+
+def test_remote_load_rejects_unknown_subop():
+    """``pld.tile.<other>`` is rejected at 3-segment dispatch."""
+    with pytest.raises(Exception, match="pld.tile"):
```

Ruff (0.15.12)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)